site/pages/rest-api/defaults.yml

```diff
@@ -26,3 +26,9 @@ nav:
 
       - name: Update Site
         href: /rest-api/examples/update-site.html
+
+      - name: Get Computers
```

Comment:
These entries would be better if put placed alphabetically. `Get Computers` should also go after `Get  Computer`